charts/helm-project-operator/values.yaml

```diff
@@ -127,7 +127,7 @@ namespaceOverride: ""
 
 image:
   repository: rancher/helm-project-operator
-  tag: v0.0.1
```

Comment:
I don't think we need to bump this tag since we are not rebuilding the image right?

Reply:
That's correct, `rancher/helm-project-operator` is not being rebuilt, only the `rancher/shell` was.